workshop_materials/cheatsheets/process_unix_cheatsheets.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euo pipefail
+
+# Note that the `unix_quick_reference.pdf` should be created manually by 
+# by exporting the `unix_quick_reference.md` as a PDF from the 
+# MacDown application on macOS _before_ running this script.
+
+# Directory where cheatsheets will be copied to
+DEST_DIR=../../docs/
```

Comment:
Let's maybe put these in a subdirectory?
```suggestion
DEST_DIR=../../docs/resources/
mkdir -p $DEST_DIR
```